### PR TITLE
Adding validate method

### PR DESCRIPTION
**Pull request by onatcipli**
Thank you for this great package but the Validate number method was missing, I added the validate method both ios and android.

### CODE REVIEW

File: lib/src/phone_number.dart [hunk 1]
```diff
@@ -54,6 +54,16 @@ class PhoneNumber {
     });
   }
 
+  Future<Map<String, dynamic>> validate(String string, String region) {
+    print('validate works');
+    print(string);
+    print(region);
```
Comment (nashfive) [inline lib/src/phone_number.dart:60]: Can you remove the print statements ?

**Review by nashfive — CHANGES_REQUESTED** [review]

File: pubspec.yaml [hunk 2]
```diff
@@ -1,7 +1,9 @@
 name: phone_number
 description: Flutter plugin for parsing, formatting and validating international phone numbers.
-version: 0.6.2+4
+version: 0.6.3
 homepage: https://github.com/nashfive/phone_number
+authors:
+  - Onat Cipli <ciplionat35@gmail.com>
```
Comment (nashfive) [inline pubspec.yaml:6]: Oh I forgot to update the authors.. 
Can you remove this for the moment? I will add the complete list in a commit.

Comment (onatcipli) [reply pubspec.yaml:6]: Okay I was going to add you but I couldn't find your email, I'm removing

File: lib/src/phone_number.dart (hunk 1 above)
Comment (onatcipli) [reply lib/src/phone_number.dart:60]: of course

**Review by nashfive — APPROVED** [review]

**Pull request merged**